packages/workbox-webpack-plugin/test/node/e2etest.js

```diff
@@ -140,6 +134,69 @@ describe(`Tests for webpack plugin`, function() {
   {
     "url": "styles/stylesheet-2.css",
     "revision": "\\w*"
+  },
+  {
+    "url": "webpackEntry.js",
+    "revision": "\\w*"
+  }
+\\]\\);`);
+      const result = regex.exec(swContents);
+      expect(result).to.exist;
+    });
+  });
+
+  it('should append publicPath to webpack generated files', () => {
+    fsExtra.copySync(
+      path.join(__dirname, '..', '..', '..', 'workbox-cli', 'test', 'static', 'example-project-1'),
+      tmpDirectory);
+
+    process.chdir(tmpDirectory);
+
+    const webpackConfig = {
+      entry: {
+        webpackEntry: path.join(tmpDirectory, 'webpackEntry.js'),
+      },
+      output: {
+        path: tmpDirectory,
+        filename: '[name].js',
+        publicPath: '/public',
```

Comment:
If you omit the trailing slash in your publicPath you should see other problems with your webpack build (not relating to workbox). If for some reason you needed to omit the trailing slash (I can't think of why you would but that doesn't mean there isn't any cause to) then this would not behave as expected and would be frustrating.

Should test different publicPaths and make sure we handle it the same way webpack does imo.